README.md

```diff
@@ -57,56 +56,68 @@ All projects must meet these minimum requirements.
   * The GPL-3.0 license will prevent your project from one day being included in
     Backdrop core, so please only use this license when necessary.
 
+
 Backdrop Contributed Project Agreement
 --------------------------------------
 
 By joining the [Backdrop Contributed Project Group](https://github.com/backdrop-contrib)
 (Backdrop Contrib, for short) you agree to the following:
 
-1. You will not push changes to a repository for which you are not a current
-   maintainer (even though joining the Backdrop Contrib group will grant you
-   access to push to any project within).
+1. You will not push changes or merge PRs for a repository for which you are not
```

Comment:
" You will not push changes, merge PRs, or create releases for a repository..."

Reply:
added. nice :)